website/docs/guides/mesh-qs.md

```diff
@@ -0,0 +1,575 @@
+
+In the command bar history, we can see that the contract failed because the status field was removed:
+<Lightbox src="/img/guides/dbt-mesh/break_contract.png" title="Broken contract" />
+
+Now, let’s walk through how model versions can be used by the Data Analytics team as they upgrade the `fct_orders` model while offering backward compatibility and a migration notice to the downstream Finance team.
+
+Rename `models/core/fct_orders.sql` to `models/core/fct_orders_v1.sql` and create a new file named `models/core/fct_orders_v2.sql`:
+
+In `models/core/fct_orders_v2.sql`, comment out `orders.status` and instead add a new field, is_return, that yields true if status = ‘returned’ and false otherwise.
+
+Then, add the following to your `models/core/core.yml` file:
+- the “is_return” column
+- the two model versions
+- a latest_version to indicate which model is the latest (and should be used by default, unless specified otherwise)
+- a deprecation_date to version 1 as well to indicate 
+
+It should now read as follows:
+<File name='models/core/core.yml'>
+
+```yml
+
+version: 2
+
+models:
+  - name: fct_orders
+    access: public
+    description: "Customer and order details"
+    latest_version: 2
+    config:
+      contract:
+        enforced: true
+    columns:
+      - name: order_id
+        data_type: number
+        description: ""
+
+      - name: order_date
+        data_type: date
+        description: ""
+
+      - name: status
+        data_type: varchar
+        description: "Indicates the status of the order"
+
+      - name: is_return
+        data_type: boolean
+        description: "Indicates if an order was returned"
+
+      - name: customer_id
+        data_type: number
+        description: ""
+
+      - name: first_name
+        data_type: varchar
+        description: ""
+
+      - name: last_name
+        data_type: varchar
+        description: ""
+
+      - name: first_order_date
+        data_type: date
+        description: ""
+
+      - name: days_as_customer_at_purchase
+        data_type: number
+        description: "Days between this purchase and customer's first purchase"
+
+    # Declare the versions, and highlight the diffs
+    versions:
+    
+      - v: 1
+        deprecation_date: 2024-06-30 00:00:00.00+00:00
+        columns:
+          # This means: use the 'columns' list from above, but exclude is_return
+          - include: all
+            exclude: [is_return]
+        
+      - v: 2
+        columns:
+          # This means: use the 'columns' list from above, but exclude status
+          - include: all
+            exclude: [status]
+
+
+```
+
+</File>
+
+Let’s verify how dbt compiles the ref statement now based on the version that we scope. Open a new file, add the following select statements, and click **Compile**.
+
+```sql
+select * from {{ ref('fct_orders', v=1) }}
+select * from {{ ref('fct_orders', v=2) }}
+select * from {{ ref('fct_orders') }}
+```
+
```

Comment:
what should they see @adinsmoor?

Reply:
I'll add another sentence to clarify what they should see: "Note how each ref is compiled to the specified version (or the latest, when not specified)."